src/main.rs

```diff
@@ -10,6 +10,7 @@ extern crate dbus;
 use std::env;
 use std::process::exit;
 use image::{load_from_memory, open};
+use image::{GenericImage, DynamicImage, FilterType};
```

Comment:
You might as well combine this use statement with the one above it (which is fetching from the same crate/module)